examples/hitl/rearrange_v2/config/language_rearrange_multi_agent_llm_gui.yaml

```diff
@@ -0,0 +1,153 @@
+  # - /habitat/task/actions@habitat.task.actions.agent_0_base_velocity: base_velocity
+  # - /habitat/task/actions@habitat.task.actions.agent_0_rearrange_stop: rearrange_stop
+
+  - /habitat/dataset/rearrangement: hssd
+  - /habitat/task/measurements:
+    - num_steps
+  - /habitat/task/lab_sensors:
+    # from previous hitl config
+    - relative_resting_pos_sensor
+    - target_start_sensor
+    - goal_sensor
+    - joint_sensor
+    - is_holding_sensor
+    - end_effector_sensor
+    - target_start_gps_compass_sensor
+    - target_goal_gps_compass_sensor
+    - localization_sensor
+    # from habitat-llm
+    - humanoid_detector_sensor
+
+  - _self_
+
+habitat:
+  task:
+    # type: RearrangeEmptyTask-v0
+    # reward_measure: num_steps
+    # success_measure: num_steps
+    # success_reward: 10.0
+    # min_distance_start_agents: 5.0
+    # slack_reward: -0.0005
+    # end_on_success: True
+    # constraint_violation_ends_episode: False
+    # constraint_violation_drops_object: True
+    # task_spec_base_path: benchmark/multi_agent/
+    # task_spec: pddl/multi_agent_tidy_house
+    # pddl_domain_def: fp
+    actions:
+      agent_1_base_velocity:
+        lin_speed: 10.0
+        ang_speed: 300
+
+    # robot_at_thresh: 3.0
+    # lab_sensors:
+    #   # Defien the human detector
+    #   humanoid_detector_sensor:
+    #     # If the human detector function is image or binary flag
+    #     return_image: False
+    #     is_return_image_bbox: False
+  gym:
+    obs_keys:
+# --- habitat-llm block
+      # - agent_0_third_rgb
+      # - agent_0_articulated_agent_arm_depth
+      # - agent_0_articulated_agent_arm_rgb
+      - agent_0_articulated_agent_arm_panoptic
+      # - agent_0_head_depth
+      # - agent_0_head_rgb
+      - agent_0_relative_resting_position
+      - agent_0_joint
+      - agent_0_ee_pos
+      - agent_0_is_holding
+      - agent_0_dynamic_obj_goal_sensor
+      - agent_0_dynamic_goal_to_agent_gps_compass
+      # dynamic_obj_start_sensor is mapped into dynamic_obj_start_sensor automatically
+      # due to sensor mapping
+      - agent_0_dynamic_obj_start_sensor
+      - agent_0_goal_to_agent_gps_compass
+      - agent_0_humanoid_detector_sensor
+      # - agent_0_articulated_agent_jaw_rgb
+      # - agent_0_articulated_agent_jaw_depth
+      - agent_0_articulated_agent_jaw_panoptic
+
+      # - agent_1_third_rgb
+      # - agent_1_articulated_agent_arm_depth
+      # - agent_1_articulated_agent_arm_rgb
+      # - agent_1_articulated_agent_arm_panoptic
+      # - agent_1_head_depth
+      - agent_1_head_rgb
+      - agent_1_head_panoptic
+      # - agent_1_relative_resting_position
+      # - agent_1_joint
+      # - agent_1_ee_pos
+      - agent_1_is_holding
+      # - agent_1_dynamic_obj_goal_sensor
+      # - agent_1_dynamic_goal_to_agent_gps_compass
+      # - agent_1_dynamic_obj_start_sensor
+# --- habitat-llm block
+  environment:
+    max_episode_steps: 750  # this is 20000 in habitat-llm
```

Comment:
We'll want to double-check this.
1. Is this enough to complete the longest tasks?
2. What's the typical step count for a normal multiplayer run? (I'll get the data)
3. What happens when this is done? We would need to call the appropriate code in HITL to trigger the next application state.